01week/rockPaperScissors.js

```diff
@@ -8,11 +25,26 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+// MY CODE //
+function rockPaperScissors(hand1, hand2){
+   if (hand1 === 'rock' && hand2 === 'rock') {
+     console.log("It's a tie!");
+   } else if (hand1 === 'paper' && hand2 === 'paper') {
```

Comment:
Could you refactor/consolidate your 3 if statements that result in "It's a tie!"? 

Hint: If hand1 and hand2 result in the same value, it's an automatic tie regardless of the string value in each hand.